source/_integrations/sia.markdown

```diff
@@ -57,4 +57,41 @@ The port used with this component must be a port no other processes use on the m
 
 ### Entities
 
-In the initial version, after setup you will see one alarm_control_panel per account and zone combination. This has a attribute for `last_heartbeat` that you could use for a template sensor in order to have that separated out. There is also a attribute for the `last_message` that will hold events happening to moisture and smoke sensors in that zone, entities supporting that will follow.
+In the initial version, after setup you will see one alarm_control_panel per account and zone combination. This entity will have 5 attributes that reflect all messages that came in for that account and zone, it includes fields for `last_code`, `zone`, `last_message`, `last_id`, `last_timestamp`. The alarm_control_panel state itself is changed based on a subset of values, including but not limited to codes: `CA`, `CB`, `CG`, `BA`, `TA`, `OA`, `NC`, `NL`, for the full list check the code on GitHub. If you expected the state to change then please log which code it was and create an issue on GitHub as well.
+
+### Events
+
+Each event that comes into your systems through SIA is also forwarded to the internal HA event bus, which can then be used to trigger automations directly on the codes that came in there. The events are created with event_type set to `sia_event_<port>_<account>`. The event_data holds many fields, see details below.
```

Comment:
Would be nice if these are available via device automation instead.

Reply:
yeah, but these aren't related to any specific device, but I'll have a look into that!